src/main.js

```diff
@@ -34,19 +34,17 @@ if (document.querySelector('.login')) {
   btnRegistration.addEventListener('click', (e) => {
     e.preventDefault();
     if (
-      expEmail.test(inputEmail.value) &&
-      expPassword.test(inputPassword.value) &&
-      expPassword.test(inputPassConfirm.value)
+      expEmail.test(inputEmail.value) && 
+      expPassword.test(inputPassword.value) && expPassword.test(inputPassConfirm.value)
     ) {
       if (inputPassword.value === inputPassConfirm.value) {
         registerUser(inputEmail.value, inputPassword.value);
+        window.location.href = '#/home';
```

Comment:
window.location.href = ''; the hash needs to be empty for redirect to the login section

Reply:
Good observation